### DISCUSSION TITLE
Question about `x-chunk` attribute in dashboard template

### DISCUSSION THREAD

**Discussion by gklandes** [shadcn-ui/ui#4661] [Q&A] [answered]
Hello,

I'm working on integrating your dashboard template into a web app I'm building with Vite. While examining the code, I noticed an attribute I'm not familiar with:

```jsx
<div className="relative hidden flex-col items-start gap-8 md:flex" x-chunk="dashboard-03-chunk-0">
```

I'm particularly curious about the `x-chunk` attribute. Could you explain its purpose and how it's used in the context of your template? 

Some specific questions I have:

1. What does the `x-chunk` attribute do?
2. Is it part of a specific framework or tool you're using?
3. Do I need to maintain this attribute when using your template in my Vite project, or can it be safely removed?

Any insights you could provide would be greatly appreciated. Thank you for your time and for creating this helpful template!

**Comment by alieldeba** [comment, marked as answer]
You can safely remove it, no worries
